lib/cucumber/rails/action_dispatch.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+ActionController::Base.class_eval do
+  cattr_accessor :allow_rescue
+end
+
+module Cucumber
+  module Rails
+    module ActionDispatch
+      module ShowExceptions
+        def call(env)
+          env['action_dispatch.show_detailed_exceptions'] = !ActionController::Base.allow_rescue
+          env['action_dispatch.show_exceptions'] = !env['action_dispatch.show_detailed_exceptions']
+          super
```

Comment:
this should be `super(env)` From previous logic

Reply:
Calling `super` without arguments passes all of the original arguments. I'm pretty sure that it will pass the modified `env` object, but given the potential for confusion here, I'll update it.

Reply:
It will normally, but because we're doing a large refactor, lets be consistent.